src/library/DIPS.Mobile.UI/Resources/Styles/ListItem/ListItemStyle.cs

```diff
@@ -0,0 +1,8 @@
+namespace DIPS.Mobile.UI.Resources.Styles.ListItem;
+
+public enum ListItemStyle
+{
+    None = 0,
+    KeyWithUnderlyingValue = 1,
```

Comment:
Jeg synes ikke disse navnene er godt nok bearbeidet, `Key` og `Value` er ikke et konsept for ListItem, så jeg tror ikke noen som er nye forstår hva dette betyr. Kanskje droppe `key` , da dette egentlig er `title` for `ListItem`. 

Noe annet jeg tenker på er om ikke dette burde dekkes med ListItem extensions i stede? Hva skiller en `style` og en `extension` her nå?